### DISCUSSION TITLE
Availability of CORE-V MCU Devkit

### DISCUSSION THREAD

**Discussion by jonnor** [openhwgroup/core-v-mcu-devkit#39] [Q&A: DevKit hardware] [answered]
Hi. From the announcement May 31, 2024 on the GroupGets page, it seems that going forward the devkit will only be available to OpenHW members. That is OK, but I would like to know roughly when the devkits might become available?

**Comment by Floadv298** [comment, marked as answer]
Hi, currently we are under re-spin, we plan to submit the files for Tape Out around November and have boards somewhen next year end of Q2,2025.  We are currently having members bi-weekly update meetings with the next 24.Jul (Wed) 11 am EST  /Flo

**Reply by jonnor** [reply]
Hi @Floadv298 - thank you for the update! Ok, that is almost 1 year away then. I guess we can expect new announcements on availability when it gets closer, and if they will be available to everyone or members-only. Best of luck with the tapeout!

**Reply by Floadv298** [reply]
Thanks!
We have some Academia Members who have Taped Out a similar design (remove FPGA, adjust PLL), technically chips are available but it's just to few and sitting at Universities.
We also do some update for the next TO for the latest Version of E40P, so it will be cool but yes, takes more time
